src/llir/raise/early.rs

```diff
@@ -741,7 +762,7 @@ fn raise_to_possibly_named_constant(names: &IdMap<i32, Sp<Ident>>, id: i32, ty_c
                 },
             }
         },
-        None => id.into(),
+        None => ast::Expr::LitInt { value: id, format: *format },
```

Comment:
having a function take `&ast::IntFormat` parameter and then doing `*` to copy it is a bit unidiomatic.  I'd suggest taking `ast::IntFormat` since it's a Copy type

Reply:
I'll freely admit that was a side effect of me having no idea which things to use for #derive on the new struct. I wrote all the code first, fixed the things that the compiler said were incorrect until only the #derive errors were left, and then just added the #derives it suggested until it worked. This probably happened because it caught the error before suggesting the copy trait.